content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -135,6 +135,73 @@ CREATE TABLE Profile (
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'planetscale']}>
+
+For the purpose of this guide, we'll use a demo SQL schema with three tables:
+
+```sql no-lines
+CREATE TABLE `Post` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `createdAt` datetime(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+  `updatedAt` datetime(3) NOT NULL,
+  `title` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `content` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `published` tinyint(1) NOT NULL DEFAULT '0',
+  `authorId` int NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `Profile` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `bio` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `userId` int NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `Profile_userId_key` (`userId`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `User` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `email` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `name` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `User_email_key` (`email`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+```
```

Comment:
Why is the SQL so different from the MySQL example?

Reply:
Not sure about the answer to this one - any idea who would be best to ask?

Reply:
Where does the SQL come from? Whoever gave it to you probably has the answer.

Reply:
The SQL is what I got from running `db push` with the schema in the 'Start from scratch' part of the tutorial, and then checking the 'Schema' tab in the PlanetScale UI. See also the SQL tab [here](https://deploy-preview-2625--prisma2-docs.netlify.app/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/using-prisma-migrate-typescript-postgres).

Reply:
Here's what I see in the PlanetScale UI:

![image](https://user-images.githubusercontent.com/3910386/148939181-6fbb462d-4e36-413a-804c-f8940b63dcd6.png)

Reply:
Ok, so the other way around. That includes quite a lot more information (collation, engine, charset) than the MySQL example but that should be fine for now.